apps/studio/components/interfaces/Organization/BillingSettings/BillingBreakdown/UpcomingInvoice.tsx

```diff
@@ -23,20 +39,26 @@ const UpcomingInvoice = ({ slug }: UpcomingInvoiceProps) => {
 
   const [usageFeesExpanded, setUsageFeesExpanded] = useState<string[]>([])
 
+  const computeCredits = useMemo(() => {
+    return (upcomingInvoice?.lines || []).find((item) =>
+      item.description.startsWith('Compute Credits')
+    )
+  }, [upcomingInvoice])
+
   const fixedFees = useMemo(() => {
     return (upcomingInvoice?.lines || [])
-      .filter((item) => !item.breakdown)
+      .filter((item) => item !== computeCredits && (!item.breakdown || !item.breakdown.length))
       .sort((a, b) => {
         // Prorations should be below regular usage fees
         return Number(a.proration) - Number(b.proration)
       })
-  }, [upcomingInvoice])
+  }, [upcomingInvoice, computeCredits])
 
   const feesWithBreakdown = useMemo(() => {
     return (upcomingInvoice?.lines || [])
-      .filter((item) => item.breakdown?.length)
+      .filter((item) => item !== computeCredits && item.breakdown?.length)
```

Comment:
Compute credits will never have a breakdown, so the first expression is not needed